add.go

```diff
@@ -69,6 +78,12 @@ func (b *Builder) Add(destination string, extract bool, source ...string) error
 		}
 	}()
 	dest := mountPoint
+
+	uid, gid, err := findUserGroupIDs(mountPoint, options.Chown)
```

Comment:
I think it'd be better to use `getUser()` from `util.go` here.

Reply:
Wow, too bad I hadn't see `getUser()` before! Thanks for pointing this out.